module/VuFind/src/VuFind/Controller/OAuth2Controller.php

```diff
@@ -212,7 +212,13 @@ public function authorizeAction()
 
         $userIdentifierField = $this->oauth2Config['Server']['userIdentifierField'] ?? 'id';
         $patron = $this->catalogLogin();
-        $patronLoginView = is_array($patron) ? null : $patron;
+        if (is_array($patron)) {
+            $patronLoginView = null;
+        } else {
+            $patronLoginView = $this->forwardTo('MyResearch', 'CatalogLogin');
```

Comment:
Is there a reason we need to call forwardTo an extra time? Could this be simplified to something like this?

```
$patronLoginView = is_array($patron) ? null : $patron;
if ($patronLoginView instanceof \Laminas\View\Model\ViewModel) {
    $patronLoginView->showMenu = false;
}
```

I'm not sure how to set things up to test this, so I wasn't able to quickly try it on my own. (I imagine that I can figure it out by looking at the unit test, so let me know if it's worth me taking the time to do that...)